crates/nu-command/src/path/mod.rs

```diff
@@ -52,19 +52,20 @@ fn err_from_value(rest: &Value, name: Span) -> ShellError {
     match rest {
         Value::Error { error, .. } => *error.clone(),
         _ => {
-            if rest.is_nothing() {
-                ShellError::OnlySupportsThisInputType {
-                    exp_input_type: "string, record or list".into(),
-                    wrong_type: "nothing".into(),
-                    dst_span: name,
-                    src_span: rest.span(),
-                }
+            let wrong_type = if rest.is_nothing() {
```

Comment:
same here, if you could give me a hint how to test it

Reply:
That seems like a weird existing choice to handle `nothing` differently. From just looking at the local context maybe addressing it at the `handle_invalid_values` call sites seems better.

Reply:
That sounds good I think.

Reply:
done